frontend/packages/core/src/AppLayout/logo.tsx

```diff
@@ -11,11 +11,13 @@ const rotate = keyframes`
 `;
 
 const StyledSvg = styled.svg`
-  height: 40px;
-  width: 40px;
+  height: 2rem;
+  width: 2rem;
+  margin-right: 1rem;
```

Comment:
In the designs I think this is split between the title and the logo. Mostly a nit for now as I don't think it'll impact anything but the margin on left and right of the title is 8px and then right margin here is 8px

Reply:
thanks for catching this. after zooming in on the design, I think the logo has a margin left and right @ 8px and the title has a margin left @ 8px.

<img src="https://user-images.githubusercontent.com/39421794/100821537-921d7780-341e-11eb-9089-d4a09afbcaec.jpg" alt="" width="400"/>

i left a to-do to remove the margin right in my PR with the V2 search bar design, since it'll be aligned to the right of the header.